doc/contributing/hw/racl/README.md

```diff
@@ -202,147 +163,177 @@ Components that bridge fabric transactions may substitute the role in incoming t
   + Pass through the role carried in the original request
   + Demote the role to a lower role
 
-## Implementation
+## Configuration
+
+RACL is configured in two ways.
+First, through a top-level configuration file, and second, through a per-instance racl-mapping configuration file.
+These are detailed below.
+The specific configuration for Darjeeling is given [here](../../../../hw/top_darjeeling/ip_autogen/racl_ctrl/doc/racl_configuration.md)
 
-This section describes the proposed tooling enhancements and microarchitectural implementation of RACL.
+### Top-level RACL Configuration
 
-### Top-level Role and Policy Mapping
+The top-level role definition and policy mapping is done via a file.
+That file is top-specific and defined by the integrator in the top-level hjson configuration file via the `racl_config` key. E.g., `racl_config: 'racl.hjson'`.
+A top-level configuration file consists of two main parts: The list of roles, as well as the specific policies for each RACL controller group. For each such group one policy must set `rot_private` to `true`, indicating that this policy is used for protecting the policy registers of that group.
+
+The following shows an example top-level configuration with a single RACL controller group called `Null`.
```

Comment:
Is `Null` special in any way? If not, I'd suggest choosing a different name that doesn't look like it has a specific meaning.

There are quite a lot of other nulls below: in each case, I really think we need to say something about the name or pick a different one.